src/ja/worker/proxy/proxy.py

```diff
@@ -4,6 +4,11 @@
 from ja.common.message.server import ServerResponse
 from ja.common.proxy.ssh import SSHConfig
 from ja.server.database.types.work_machine import WorkMachineResources
+from ja.worker.message.register import RegisterWorkerCommand
+from ja.worker.message.retire import RetireWorkerCommand
+from ja.worker.message.done import JobDoneCommand
+from ja.worker.message.crashed import JobCrashedCommand
+from ja.server.database.database import ServerDatabase
 
 
 class IWorkerServerProxy(ContinuousProxy, ABC):
```

Comment:
You are not supposed to implement the actual proxy class.
You are supposed to create a proxy dummy in the test package that mimics the behavior of the actual server without being functional.
This dummy must **not** depend on the implementation of the database.
It should keep track of jobs with something like a dictionary instead.